cmd/bridge/main.go

```diff
@@ -14,15 +14,17 @@ import (
 	"strings"
 
 	authopts "github.com/openshift/console/cmd/bridge/config/auth"
+	"github.com/openshift/console/cmd/bridge/server"
```

Comment:
Cant we have the this new `server` part of the the original one? So we can avoid name collisions.

Reply:
I did struggle with this a bit. In my opinion, the original server package is misnamed. Its concern is constructing an HTTP handler, not a server. I debated adding the new HTTP server logic to the existing server package. Ultimately, I decided against it because it would likely involve refactoring that I thought was out of scope for a bug fix. Would it be enough to add a TODO comment to resolve this name conflict in a more meaningful way?